Resources/Prototypes/_Evin/Entities/Player/silicon.yml

```
@@ -0,0 +1,41 @@
+- type: entity
+  id: PlayerBorgMedical
+  name: Медицинский киборг
```

опять name с большой. увы
```suggestion
  name: медицинский киборг
```